tests/integration/models/setup_utils.py

```diff
@@ -120,6 +109,40 @@ def mesh_arguments(self) -> dict:
         return mesh_sizes
 
 
+class WellGeometryMixin:
+    """Mixin class for models with wells."""
+
+    nd: int
+    """Number of dimensions."""
+    params: dict
+    """Model parameters."""
+
+    def set_well_network(self) -> None:
+        """Assign well network class."""
+        num_wells = self.params.get("num_wells", 1)
+        if self.nd == 2:
+            # Comments are the intersection with fractures in
+            # RectangularDomainOrthogonalFractures
+            wells = [
+                pp.Well(np.array([0.5, 0.1])),  # Intersects one fracture
```

Comment:
Is the coordinate a 1d or 2d (2x1) array for a point well?

Reply:
@IvarStefansson: Can you handle this one?

Reply:
Not sure. I'm a bit careless when it comes to that distinction. To my occasional regret...

Reply:
I think we should be consistent, or else things will get very confusing.

Reply:
According to the documentation of the `Well` class, a point well should be defined as a 3x1 array. I fixed their definitions.